src/handlers/shared/check-task-stale.ts

```diff
@@ -0,0 +1,12 @@
+export function checkTaskStale(staleTask: number, createdAt: string) {
+  let days: number | undefined;
+  let staleToDays: number | undefined;
+  let isTaskStale = false;
+
+  if (staleTask !== 0) {
+    days = Math.floor((new Date().getTime() - new Date(createdAt).getTime()) / (1000 * 60 * 60 * 24));
+    staleToDays = Math.floor(staleTask / (1000 * 60 * 60 * 24));
+    isTaskStale = days >= staleToDays;
+  }
+  return isTaskStale;
+}
```

Comment:
Doesn't make sense to allow `| undefined`

Just declare it inside of the if block if thats the only place its used.